src/view/lesson_environment.js

```diff
@@ -130,33 +133,71 @@ var LessonEnvironment = React.createClass({
     this._startStep(0);
   },
 
+  _closeSidebar: function() {
+    this.setState({docked: false});
+  },
+  
+  _openSidebar: function() {
+    this.setState({docked: true});
+  },
+
+  showButton: function() {
+    if (!this.state.docked) {
+      return <div style={{width: "20px",
+                          float: "right",
+                          backgroundColor: '#03a9f4'}}>
+               <a href='#'
+                  onClick={this._openSidebar}
+                  style={{textDecoration: 'none', color: 'white'}}> 
+                 C
```

Comment:
Can you try using a CSS transform to rotate this text instead? If it doesn't work well (or if it's too much work to align it properly), we can leave it like this for now. Here is an example you can try to adapt:

```html
<!doctype html>
<html>
  <body>
    <div style="border: 3px solid black; background-color: orange; margin: 50px; padding: 50px">
      <p>COMANDOS</p>
    </div>
    <div style="border: 3px solid black; background-color: orange; margin: 50px; padding: 50px">
      <p style="trasnform-origin: center center; width: 8em; transform: rotate(90deg);">COMANDOS</p>
    </div>
  </body>
</html>
```

Don't worry too much about this, though. If it doesn't work, can you split the word using JavaScript instead? Like putting `"COMANDOS".split("").join("\n")` instead of all those lines.